pennylane/qnn/torch.py

```diff
@@ -449,19 +451,16 @@ def construct(self, args, kwargs):
     def __getattr__(self, item):
         """If the given attribute does not exist in the class, look for it in the wrapped QNode."""
         if self._initialized:
-            return getattr(self.qnode, item)
+            with contextlib.suppress(AttributeError):
+                return getattr(self.qnode, item)
 
-        try:
-            return self.__dict__[item]
-        except KeyError as exc:
-            raise AttributeError(item) from exc
+        return super().__getattr__(item)
 
     def __setattr__(self, item, val):
         """If the given attribute does not exist in the class, try to set it in the wrapped QNode."""
-        if self._initialized:
+        if self._initialized and item in self.qnode.__dict__:
             setattr(self.qnode, item, val)
-        else:
-            self.__dict__[item] = val
+        return super().__setattr__(item, val)
```

Comment:
I like this standard! maybe we should only assign to the Layer if we don't assign to the qnode in this new scenario.
```suggestion
        else:
            super().__setattr__(item, val)
```
also needs a docstring update